src/inc/quic_platform.h

```diff
@@ -80,7 +80,7 @@ typedef struct CXPLAT_SLIST_ENTRY {
 #define QUIC_POOL_WORKER                    'A0cQ' // Qc0A - QUIC Worker
 #define QUIC_POOL_LISTENER                  'B0cQ' // Qc0B - QUIC Listener
 #define QUIC_POOL_CID                       'C0cQ' // Qc0C - QUIC CID
-#define QUIC_POOL_CIDHASH                   'D0cQ' // Qc0D - QUIC CID Hash
+#define QUIC_POOL_CIDSLIST                  'D0cQ' // Qc0D - QUIC CID Hash
```

Comment:
Instead of replacing this and adding it down at the bottom, can you leave this as is, and put the new one and the end?